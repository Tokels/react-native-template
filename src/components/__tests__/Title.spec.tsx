import * as React from 'react';
import { expect, it } from '@jest/globals';
import renderer from 'react-test-renderer';
import { Text } from 'react-native';

it('renders correctly', () => {
  const tree = renderer.create(<Text>Snapshot test!</Text>).toJSON();

  expect(tree).toMatchSnapshot();
});
