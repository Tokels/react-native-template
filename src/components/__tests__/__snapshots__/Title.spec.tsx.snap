// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text>
  Snapshot test!
</Text>
`;
